torch/_refs/__init__.py

```diff
@@ -1879,6 +1880,39 @@ def sum(
     )
 
 
+def _is_expandable_to(shape, desired):
+    # Python implementation of
+    # aten/src/ATen/ExpandUtils.h:is_expandable_to
+    if len(shape) > len(desired):
+        return False
+    for i in range(len(shape)):
+        if shape[i] != desired[i] and shape[i] != 1:
```

Comment:
Doesn't this need to go from right-to-left?

Reply:
Yes, it should be right-to-left, I wonder how tests pass 🤔